libraries/imageslider/src/main/java/com/trendyol/uicomponents/imageslider/CVFloatingZoomView.java

```diff
@@ -0,0 +1,356 @@
+package com.trendyol.uicomponents.imageslider;
```

Comment:
For supporting fast build, fully kotlin project etc, should use only kotlin files